fl4health/preprocessing/warmed_up_module.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+from logging import INFO
+from pathlib import Path
+from typing import Optional
+
+import torch
+from flwr.common.logger import log
+
+
+class WarmedUpModule:
+    """This class is used to load a pretrained model into the current model."""
```

Comment:
Super minor, but I might say `into a target model` since we provide a target into which, we insert the weights.